packages/contracts-bedrock/test/L2/L2StandardBridge.t.sol

```diff
@@ -131,18 +131,18 @@ contract L2StandardBridge_Test is Bridge_Initializer {
         assertEq(address(l2ToL1MessagePasser).balance, 100);
     }
 
-    /// @dev Tests that the receive function reverts with custom gas token.
-    function testFuzz_receive_customGasToken_reverts(uint256 _value) external {
-        vm.prank(alice, alice);
-        vm.mockCall(address(l1Block), abi.encodeWithSignature("gasPayingToken()"), abi.encode(address(1), uint8(2)));
-        vm.deal(alice, _value);
-        (bool success, bytes memory data) = address(l2StandardBridge).call{ value: _value }(hex"");
-        assertFalse(success);
-        assembly {
-            data := add(data, 0x04)
-        }
-        assertEq(abi.decode(data, (string)), "StandardBridge: cannot bridge ETH with custom gas token");
-    }
+    // /// @dev Tests that the receive function reverts with custom gas token.
```

Comment:
Rather than disabling this test, couldn't we rewrite it to assert the behavior we have implemented?  Same goes for the subsequent tests.

Reply:
Added!